tencentcloud/resource_tc_instance_test.go

```diff
@@ -825,3 +854,51 @@ resource "tencentcloud_instance" "sg" {
 		rule,
 	)
 }
+
+func testAccInstanceConfigWithInstanceTypeChanged(name string) string {
```

Comment:
it would be better if the name is corresponding to its functionality, it actually changes the hostname to force create a new instance, not instance type.